apps/platform/src/app/helpers/projectHelper.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Determines the background color class for a project's progress bar
+ * based on the given progress percentage.
+ *
+ * @param {number} progressPercentage - The current progress of the project as a percentage (from 0 to 100).
+ * @returns {string} - A string representing the CSS class name for the background color.
+ *
+ * - Returns `'bg-[#C9767B]'` for progress between 1% and 33%.
+ * - Returns `'bg-[#E2CB55]'` for progress between 34% and 66%.
+ * - Returns `'bg-[#09CE78]'` for progress between 67% and 100%.
+ * - Returns `'bg-green'` if progress is 0% or any other value outside the 1-100 range.
```

Comment:
Maybe add these to tailwind config so we can do eg.
```suggestion
 * - Returns `'bg-progressLow'` for progress between 1% and 33%.
 * - Returns `'bg-progressMedium'` for progress between 34% and 66%.
 * - Returns `'bg-progressHigh'` for progress between 67% and 100%.
 * - Returns `'bg-green'` if progress is 0% or any other value outside the 1-100 range.
```